birdie/src-tauri/src/config/router.rs

```diff
@@ -16,6 +15,7 @@ use ethos_core::types::repo::CloneRequest;
 use {crate::DEFAULT_DRIVE_MOUNT, ethos_core::utils, std::path::Path};
 
 use crate::state::AppState;
+use crate::types::config::AppConfig;

```

Comment:
should this be `birdie::types::config::AppConfig` or `birdie::types::config::BirdieConfig`

for clarity I prefer `BirdieConfig`

thoughts @rudoi ?

Reply:
i agree